sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -104,9 +104,21 @@ def run_pipeline(self, pipeline):
     prepare_response = job_service.Prepare(
         beam_job_api_pb2.PrepareJobRequest(
             job_name='job', pipeline=proto_pipeline))
+    if prepare_response.staging_session_token:
+      # Must commit an empty manifest.
+      artifact_service = beam_artifact_api_pb2_grpc.ArtifactStagingServiceStub(
+        grpc.insecure_channel(prepare_response.artifact_staging_endpoint.url))
+      commit_manifest = artifact_service.CommitManifest(
+          beam_artifact_api_pb2.CommitManifestRequest(
```

Comment:
So just to confirm, the intention is to stage an empty manifest for now until we are ready to wire in actual artifact staging in the python sdk?  If so, then I think we should add a TODO line there for clarity.

Reply:
Done.